core-web/src/main/java/org/silverpeas/core/web/mvc/route/ComponentRequestRouter.java

```diff
@@ -23,9 +23,14 @@
  */
 package org.silverpeas.core.web.mvc.route;
 
+import org.silverpeas.core.admin.component.model.ComponentInst;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.silverpeas.core.admin.component.model.ComponentInst'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)